cmd/podman/containers/ps.go

```diff
@@ -242,7 +243,8 @@ func ps(cmd *cobra.Command, _ []string) error {
 	defer w.Flush()
 
 	headers := func() error { return nil }
-	if !(listOpts.Quiet || cmd.Flags().Changed("format")) {
+	noHeading, _ := cmd.Flags().GetBool("noheading")
+	if !noHeading && !(listOpts.Quiet) {
```

Comment:
Isn't this missing the old check for the `--format` flag?

Reply:
oops you are right.  FIxed.